Cqlmapper support

pacejackson opened the pull request.

spladug commented on baseplate_cookiecutter/{{cookiecutter.project_slug}}/README.md:
```diff
@@ -5,14 +5,21 @@ This is the {{ cookiecutter.project_slug }} service. It is built on [Baseplate].
 
 {% if cookiecutter.integrations.sqlalchemy %}
-## Database Initialization
+## Sqlalchemy Database Initialization
```
"Relational" ?

spladug commented on baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/cql.py:
```diff
@@ -0,0 +1,35 @@
+
+
+class MyModel(Model):
+
```
extra blank here

spladug commented on baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/cql.py:
```diff
@@ -0,0 +1,35 @@
+    id = columns.UUID(primary_key=True, default=uuid.uuid4)
+    name = columns.Text()
+    value = columns.Integer()
+
```
not enough blanks here

spladug commented on baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/sql.py:
```diff
@@ -1,3 +1,4 @@
+
```
extra blank here?

pacejackson commented:
:haircut_man: Addressed feedback.  /models does have an `__init__.py`
https://github.com/reddit/baseplate-cookiecutter/pull/25/files#diff-c3a60d2f395bc3efc9cd8e49a2560c92
